tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -310,6 +329,38 @@ def test_learning_pipelining_without_interleaving_ucc_for_p2p(self):
     def test_inference_pipelining_without_interleaving_ucc_for_p2p(self):
         self._test_hybrid_backends(True)
 
+    @unittest.skipUnless(CAN_SKIP_SYNC_AFTER_BATCH_ISEND_IRECV, "Requires https://github.com/pytorch/pytorch/pull/82450")
+    def test_learning_pipelining_without_interleaving_skyp_sync_after_batch_isend_irecv(self):
+        self.test_learning_pipelining_without_interleaving(sync_batch_comm=False)
+
+    @unittest.skipUnless(CAN_SKIP_SYNC_AFTER_BATCH_ISEND_IRECV, "Requires https://github.com/pytorch/pytorch/pull/82450")
+    def test_inference_pipelining_without_interleaving_skip_sync_after_batch_isend_irecv(self):
+        self.test_inference_pipelining_without_interleaving(sync_batch_comm=False)
+
+    @unittest.skipUnless(CAN_SKIP_SYNC_AFTER_BATCH_ISEND_IRECV, "Requires https://github.com/pytorch/pytorch/pull/82450")
+    def test_learning_async_pipelining_without_interleaving_skip_sync_after_batch_isend_irecv(self):
+        self.test_learning_async_pipelining_without_interleaving(sync_batch_comm=True)
```

Comment:
Shouldn't be here `sync_batch_comm=False`?

Reply:
yeah, thanks. this pr is a well-balanced task which made me consider as a routine... :/